internal/server/http.go

```diff
@@ -45,7 +46,9 @@ func handleUX() http.Handler {
 // healthy (can connect to the Perseus database) and '500 Internal Server Error' if not
 func handleHealthz(db store.Store) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if err := db.Ping(r.Context()); err != nil {
+		ctx, cancel := context.WithTimeout(r.Context(), 300*time.Millisecond)
```

Comment:
aside: I could imagine a case where the DB intermittently fails to respond in ~300ms. I guess it depends on your infrastructure and its healthcheck failure policy, but hopefully a single failure here wouldn't e.g. kill the pod. Not sure if it warrants a configurable duration.

Reply:
That's a fair point.  Wasn't sure what a "reasonable" value would be but it should have _some_ timeout.  Wouldn't be hard to make it configurable if we want to go that path.  It's also straightforward to configure K8S to require multiple failed health checks before killing the pod. 🤷

Reply:
@bndw I made this configurable via a `HEALTHZ_TIMEOUT` env var, with a default of 300ms.